src/main/java/net/sf/jabref/event/AddOrChangeEntryEvent.java

```diff
@@ -0,0 +1,28 @@
+package net.sf.jabref.event;
+
+import net.sf.jabref.model.entry.BibEntry;
+
+public class AddOrChangeEntryEvent implements Event {
```

Comment:
Do not introduce this event. -> No hierarchy.

If one wants to listen to both Add and Change, then one implements two subscribe methods internally calling a generic method.

This makes it much more easier for the publisher of the event. In your case, if one adds an entry, one has to call both, AddEntryEvent and AddOrChangeEntryEvent.